tests/fixture/tmpnet/defaults.go

```diff
@@ -50,6 +50,7 @@ func DefaultFlags() FlagsMap {
 		config.LogDisplayLevelKey:               logging.Off.String(), // Display logging not needed since nodes run headless
 		config.LogLevelKey:                      logging.Debug.String(),
 		config.MinStakeDurationKey:              DefaultMinStakeDuration.String(),
+		config.ProposerVMUseCurrentHeightKey:    true,
```

Comment:
This was the main fix for the e2e test. The reason that we need to issue the transfer tx on the destination chain is to update the pChainHeight on the destination subnet to enable verifying the warp message. However, without the flag set, the P-chain height lags by `2` minutes and at least `16` blocks. During the test, the P-chain has height 9. So, even if we added a 2 minute sleep, it wouldn't advance past height=0 (which is before the subnet validators are added).

Reply:
It's valid to use in a production network.. But not ideal